pkg/rewards/operatorPISplitSnapshots.go

```diff
@@ -0,0 +1,89 @@
+package rewards
+
+const operatorPISplitSnapshotQuery = `
+WITH operator_pi_splits_with_block_info as (
+	select
+		ops.operator,
+		ops.activated_at::timestamp(6) as activated_at,
+		ops.new_operator_avs_split_bips as split,
+		ops.block_number,
+		ops.log_index,
+		b.block_time::timestamp(6) as block_time
+	from operator_pi_splits as ops
+	left join blocks as b on (b.number = ops.block_number)
```

Comment:
Same as the comment above; should probably make this an inner join

Reply:
Done in [64bf833](https://github.com/Layr-Labs/sidecar/pull/117/commits/64bf8334c40fcf4505f74672a8341a96cd9858a2)